src/index.tsx

```diff
@@ -197,6 +202,7 @@ class Cropper extends React.Component<CropperProps, State> {
     const { crop, zoom } = getInitialCropFromCroppedAreaPixels(
       initialCroppedAreaPixels,
       this.mediaSize,
+      rotation - initialRotation,
```

Comment:
Why do you need to substract `initialRotation` from `rotation`?

Reply:
Regarding substraction ```initialRotation``` from ```rotation``` it's part of the logic of rotation and it might seem slightly tricky but it necessary to handle the angle at which the position was changed before save it to local storage or elsewhere. 

Deeper explanation: imagine we rotated image 90 degrees, changed position and save the area and the angle to store, after page reload we use the saved values to initialize cropper - it's need to set ```initialRotation``` to 90 and ```rotation``` to 0, it will work fine, ok... But then we decide to rotate image(using ```rotation``` prop) to 90 again - this is where the problem lies. ```initialRotation``` shouldn't apply in case when ```rotation``` equal to original angle(when the position was changed before saving) cause we already have the correct area for this angle.

Simply put, it's required to depreciate the effect when second time the same angle will be applied. 
I know it's tangled and I wrote not the best explanation, sorry for that :)

About parent rotation - it's really up to you, I just suggested the improvement cause I personally encountered with this problem that's why I thought it might be useful for other users.

Reply:
I just noticed something: can't we only use the `rotation` prop here?
If it's correctly set on load it would avoid the issue you mentioned I think.

Reply:
I got you opinion, but I don't have idea how to do it without extra prop. Since the rotation formula was apllied to all points, we need to know what the angle it was when the position was exported - it's kind of "anchor" for further transformation. This is the reason why we need new one prop ```initialRotation``` 

Otherwise it's need to think on another solution if we want to avoid additional prop

Reply:
My point is about initialising `rotation` with the value coming from local storage. I think it should do the job. However, the logic in `getInitialCropFromCroppedAreaPixels` should be adjusted to stop doing `- rotation`

Reply:
Another solution could be wrap the image with div container and apply rotation to it. But it can lead to other bugs and unexpected dragging behavior

Reply:
> 
> 
> My point is about initialising `rotation` with the value coming from local storage. I think it should do the job. However, the logic in `getInitialCropFromCroppedAreaPixels` should be adjusted to stop doing `- rotation`

I've tried to do it in such way but id didn't work. I'll try to come up with it again, but I'm afraid it could be so complex

Reply:
I just tried it out and now I'm actually wondering if we even need a change: I tried storing the rotation in the local storage with the latest version and it looks to be working fine to me: https://codesandbox.io/s/react-easy-crop-demo-with-local-storage-73u6i?file=/src/index.js

Could you try it please and tell me if I'm missing something?

Reply:
Hmm, you know, it works well on your demo. Looks like we caught some misunderstanding. Cause the person who opened that issue probably didn't apply the rotation when set ```initialCroppedAreaPixels```. From my side I considered the issue like "if image was rotated for ex. 90 deg, then shifted, and then during the next initialization with previous crop area and any other custom angle, the image should take the same position related to container but with different angle"

Reply:
So let's deal only with ```parentRotationAngle``` I believe it could be helpful in some cases. For example to develop some constructor or wysiwyg editor, etc. What you think about it?

Reply:
Thanks for checking the example, I'll share it on the original issue then.

Regarding `parentRotationAngle`, I am not a big fan of it to be honest, it looks really specific. Moreover, it doesn't cover all use cases: for example, when zooming with the mouse, the position is not correct anymore.

In order to be more generic, I would rather have a callback option like `getCoordinates()` to provide inversion-of-control. Thanks to it, the user would be able to do any kind of logic before the coordinates are used inside the library. What do you think?

Reply:
Got it, no problem then :)
Obviously it might be useful for some cases, in general this is cool idea.
But now I'm not sure if that will help to fix dragging distortion when parent container was rotated or maybe even translated.

In order to not deliver not conceptual solutions in codebase, I can use raw code of the library in my project and modify it as I want to achieve needed effect, right?

Reply:
Yes forking is always an option :)

Reply:
So I think we can close this PR?

Reply:
Yes I think this is better.

Thanks for the work and the nice communication. I hope you can contribute again in the future and have it merged :)

Reply:
Thank you too. Maybe we would work in the future again. Have a nice coding )